### PR TITLE
Adding SGE Chain

### PR DESCRIPTION
**Pull request by shang-chi-7**
## Description

Adding SGE Chain

### Adding Chains

If adding a new chain, please ensure the following:
- [x] Chain is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).

  Note:
   - Chain's registration must have `staking` defined, with at least one `staking_token` denom specified.
   - Chain's registration must have `fees` defined; at least one fee token has low, average, and high gas prices defined.
- [x] Add chain to bottom of `zone_chains.json`

  Note:
   - RPC and REST must not have any CORS blocking of the Osmosis domain.
   - RCP node must have WSS enabled.

### Adding Assets

If adding a new asset, please ensure the following:
- [x] Asset is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
- [x] Add asset to bottom of `zone_assets.json`.

  Note:
   - The IBC channel referenced in `path` must be registered to the Chain Registry.
   - `osmosis_main` defaults to `false` (or else cite the listing rule that justifies enlisting to the Main app)

### On-chain liquidity

If adding a new asset, please provide the plan for on-chain liquidity of the asset: (choose one)
- [ ] The submitting team will ensure a pool will be created soon. Please hold off until further communication, which will notify of the Pool ID. (Please set the PR's status to Draft until the pool has been created)

OR
- [x] A preview link is requested for the submitting team to be able to create a pool using the Osmosis Frontend.

OR
- [ ] The token is, or will be, going through a StreamSwap stream; thus, the token should be listed without requiring on-chain liquidity. A Pool ID will be provided in this PR following the stream's completion once the team has had a chance to create a pool using the earned funds.

### Validation Testing

If adding or updating a chain or asset, the changes must be validated from the updated build of the Osmosis Frontend. Testers will need a small amount of tokens to validate. 
  - [ ] Validaters can buy a small amount of this asset from Pool ID: {PROVIDE POOL ID}.
  - [ ] Preview build has been validated.

### CODE REVIEW

**Comment by reeves89** [comment]
@JeremyParish69 Please review the pr.

File: osmosis-1/osmosis.zone_assets.json [hunk 1]
```diff
@@ -1553,6 +1553,13 @@
       "base_denom": "erc20/0x655ecB57432CC1370f65e5dc2309588b71b473A9",
       "path": "transfer/channel-204/erc20/0x655ecB57432CC1370f65e5dc2309588b71b473A9",
       "osmosis_verified": false
+    },
+    {
+      "chain_name": "sge",
+      "base_denom": "usge",
+      "path": "transfer/channel-5485/usge",
+      "osmosis_verified": true,
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:1561]: Please default osmosis_verified to false, and please remove osmosis_fronteir, which isn't used anymore.

Comment (shang-chi-7) [reply osmosis-1/osmosis.zone_assets.json]: @JeremyParish69 Please review again

**Comment by JeremyParish69** [comment]
Thanks @shang-chi-7 The fixes appear correct and will get a preview link for you once another blocking issue has been resolved.

**Comment by reeves89** [comment]
> Thanks @shang-chi-7 The fixes appear correct and will get a preview link for you once another blocking issue has been resolved.

@JeremyParish69 Possible to provide any expected date?

**Comment by JeremyParish69** [comment]
> > Thanks @shang-chi-7 The fixes appear correct and will get a preview link for you once another blocking issue has been resolved.
> 
> @JeremyParish69 Possible to provide any expected date?

Apologies for the delay--I don't have any date for the resolution as I don't see it planned, but I imagine it should be within the next week.

**Comment by JeremyParish69** [comment]
@shang-chi-7 you may use this preview build to deposit SGE and create a liquidity pool using the Osmosis Zone frontend. 
https://osmosis-frontend-git-jp69-asset-sge-add-osmo-labs.vercel.app/assets
Please provide the SGE pool id.
cc @reeves89

**Comment by reeves89** [comment]
> @shang-chi-7 you may use this preview build to deposit SGE and create a liquidity pool using the Osmosis Zone frontend. https://osmosis-frontend-git-jp69-asset-sge-add-osmo-labs.vercel.app/assets Please provide the SGE pool id. cc @reeves89

Pool Id: Pool #1233 (Link: https://osmosis-frontend-git-jp69-asset-sge-add-osmo-labs.vercel.app/pool/1233) @JeremyParish69

**Comment by reeves89** [comment]
@JeremyParish69 Thank you!! 
We are not able to find the token in the Osmosis app asset list(https://app.osmosis.zone/assets).

**Pull request merged**